README.md

```diff
@@ -786,6 +786,49 @@ We need to create an HTTP factory by using the `createHttpFactory()` function, p
 - `get()` - A proxy for Angular `TestBed.get()`
 - `expectOne()` - Expect that a single request was made which matches the given URL and it's method, and return its mock request
 
+Spectator also makes testing concurrent requests easier. Similar to `spectator.expectOne()`, you can use,
```

Comment:
Thanks for the effort, but I prefer to combine this example with the previous one. Please add the relevant code to previous snippet, and leave the method and the description here. Thanks.